Security/Security.Platform.Android/Checks/BackupAllowedCheck.cs

```diff
@@ -7,22 +7,22 @@ namespace AeroGear.Mobile.Security.Checks
     /// <summary>
     /// Check to determine whether the allowBackup flag is enabled for the application.
     /// </summary>
-    public class BackupDisallowedCheck : AbstractSecurityCheck
+    public class BackupAllowedCheck : AbstractDeviceCheck
```

Comment:
should this be renamed to `BackupEnabledCheck`?

Reply:
👍